scripts/demo.py

```diff
@@ -0,0 +1,34 @@
+from matplotlib import pyplot as plt
+import numpy as np
+
+import genome_browser as gb
+
+n = 105  # Length of random genomic interval.
+
+g = gb.GenomeDiagram()
+
+# Plot a density of random data, interpolated and filled.
+graph_track = gb.GraphTrack("Random Density")
+graph_track.add_graph(
+    gb.Graph(x=np.arange(n), y=np.abs(np.random.randn(n)), fmt="interpolate", fill=True)
+)
+g.add_track(graph_track)
+
+# Plot 9 random intervals.
+interval_track = gb.IntervalTrack("Random Intervals", height_ratio=0.4, step=50)
+for _ in range(9):
+    interval = gb.Interval(
+        chrom="chr3",
+        start=np.random.randint(0, n),
+        end=np.random.randint(0, n),
+        strand=np.random.choice(["+", "-", "."]),
+        color=np.random.choice(["#E74C3C", "#3498DB", "0.2"]),
+    )
+    interval_track.add_interval(interval)
+g.add_track(interval_track)
+
+# Annotate the figure with interval specific metadata. Will always appear in lower-right
+g.annotation = "{}:{:,}-{:,}".format("chr3", 20000, 812383)
+
+fig, axes = g.draw()
+plt.show()
```

Comment:
It's nice to see the example right away :slightly_smiling_face:

Reply:
#removing`setup.py` (below :point_down:)

I haven't seen `setuptools` in production like ever :smile: (worked with Python for the past 3 years). Just `pipenv` and lately `poetry`. And "always write as if for production", goes the dev saying.

If you'd rather keep it, sure, but it's always a PITA to keep multiple points updated, and - in my experience - never works.